### PR TITLE
Expands Pickup Times With More Flexibility and Fixes Closed Dates Showing Up As Future Selections.

### PR DESCRIPTION
**Pull request by timnolte**
* Adds npm modules for grunt tasks.
* Fixes pickup time so that closed dates are excluded, not just today.
* Changes days ahead to a text field with HTML5 number restrictions/validations to allow for maximum flexibility and reduced internationalization needs.
* Changes date/time selection to include day of week and date based on WordPress date format.
* Adds 36 hour delay option.
* Adds Tim Nolte(tnolte) to constributors list in README.
* Updates version numbers.
* Updates langauge files.

### CODE REVIEW

**Comment by vyskoczilova** [comment]
@mattbanks if you will accept this pull, ping me, I will update the Czech translation

**Comment by mattbanks** [comment]
This PR looks good! There are some indentation issues in a couple of PHP files. @timnolte can you update those and then I'll merge? @vyskoczilova when this gets merged, you can submit a PR with Czech and I'll cut a new release!

**Comment by timnolte** [comment]
@mattbanks this PR should be ready to go now. Thanks!

File: public/class-local-pickup-time.php [hunk 1]
```diff
@@ -343,60 +347,73 @@ public function create_hour_options() {
...
-							$tNow = $todayStart;
-						}
+						$todayStart = $tStart;
+						$delayStart = strtotime("+$delay_minutes minutes", $current_time);
+
+						while ( $todayStart <= $delayStart ) {
+
+							$todayStart = strtotime("+$interval minutes", $todayStart);
+
+            }
```
Comment (mattbanks) [inline public/class-local-pickup-time.php]: indent matching here please :)

**Comment by timnolte** [comment]
@mattbanks change made. Thanks!

**Comment by mattbanks** [comment]
awesome! merging this now, but I'm heading out for a family thing for a few days and I don't have SVN setup on my current machine. @vyskoczilova do you want to submit a PR with the translations? I'll be back on Tuesday and can merge and cut a new release.

**Pull request merged**